crates/networking/p2p/rlpx/eth/transactions.rs

```diff
@@ -165,8 +165,8 @@ impl RLPxMessage for NewPooledTransactionHashes {
 pub(crate) struct GetPooledTransactions {
     // id is a u64 chosen by the requesting peer, the responding peer must mirror the value for the response
     // https://github.com/ethereum/devp2p/blob/master/caps/eth.md#protocol-messages
-    id: u64,
-    transaction_hashes: Vec<H256>,
+    pub(crate) id: u64,
+    pub(crate) transaction_hashes: Vec<H256>,
```

Comment:
can we create a helper function and keep these private? So that we don't break encapsulation